Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -13,10 +13,11 @@ public class ProjectGenerator {
         self.project = project
     }
 
-    public func generateXcodeProject() throws -> XcodeProj {
+    public func generateXcodeProject(to projectDirectory: Path? = nil) throws -> XcodeProj {
```

Comment:
Is this default argument necessary? It seems there is no use of the default value.

Reply:
Keep in mind that XcodeGenKit is used as a library in other third party tools

Reply:
Oh I overlooked `public` modifier.